.lgtm/cpp-queries/fgets.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Use of fgets()
+ * @description fgets() is dangerous to call. Use read_line() instead.
+ * @kind problem
+ * @problem.severity error
+ * @precision high
+ * @id cpp/fgets
+ * @tags reliability
+ *       security
+ */
+import cpp
+
+
+predicate dangerousFunction(Function function) {
```

Comment:
hmm, double empty line? that's typically no what we do in our codebase, but i guess doesn't matter